app/models/setting/remote_execution.rb

```diff
@@ -22,7 +22,10 @@ def self.load_defaults
                  'root'),
         self.set('remote_execution_effective_user_method',
                  N_('What command should be used to switch to the effective user. One of %s') % SSHExecutionProvider::EFFECTIVE_USER_METHODS.inspect,
-                 'sudo'),
+                 'sudo',
+                 'remote_execution_effective_user_method',
+                 nil,
+                 { :collection => Hash[SSHExecutionProvider::EFFECTIVE_USER_METHODS.map{|method| [method, method]}]}),
```

Comment:
It's explained in the issue http://projects.theforeman.org/issues/14963.  This may be static, but everything needs to be wrapped in proc, your change doesn't work against the latest foreman develop. It generates an exception when you load the page.